iceoryx_hoofs/platform/CMakeLists.txt

```diff
@@ -17,7 +17,20 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-if(LINUX)
+set(IOX_VERSION_STRING "2.90.0")
+project(iceoryx_platform VERSION ${IOX_VERSION_STRING})
+
```

Comment:
I think you will need something like this later for the colcon build to build automatically shared libraries:

```
if(CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES Darwin)
    option(BUILD_SHARED_LIBS "Create shared libraries by default" ON)
endif()
```